sample/GoogleIO17/sponsors

```diff
@@ -1 +1,155 @@
-[]
\ No newline at end of file
+[
```

Comment:
@mariobehling I don't think we should assume a list of sponsors when they haven't officially mentioned them. If there are no sponsors listed on the official event website, then we shouldn't be showing anything either. 

cc @arp95